src/Models/FillsListModel.vala

```diff
@@ -0,0 +1,70 @@
+/*
+* Copyright (c) 2019 Alecaddd (http://alecaddd.com)
+*
+* This file is part of Akira.
+*
+* Akira is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 3 of the License, or
+* (at your option) any later version.
+
+* Akira is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+* GNU General Public License for more details.
+
+* You should have received a copy of the GNU General Public License
+* along with Akira.  If not, see <https://www.gnu.org/licenses/>.
+*
+* Authored by: Giacomo "giacomoalbe" Alberini <giacomoalbe@gmail.com>
+*/
+
+
+public class Akira.Models.FillsListModel : GLib.Object, GLib.ListModel {
+    private GLib.List<Akira.Models.FillsItemModel> fills_list;
+
+    public FillsListModel () {
```

Comment:
There's no need to declare the object construct if you're not doing anything with it.